NVDAAddonTemplate/__init__.py

```diff
@@ -118,7 +118,8 @@ def getInput(prompt, default = ""):
 	answer = ""
 	while answer == "":
 		print(prompt)
-		answer = input(default)
+		default_prompt = "The default value is:(" + default + ")Please enter your choice"
```

Comment:
Would something like this be a better choice?

Default: {default}
Please enter a value, or press enter to accept the default.

Also, please use a format string like this. it's easier if we decide to translate, and cleaner for changing later.

`		default_prompt = "The default value is: {default} Please enter your choice".format(default)`